trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/add-to-an-existing-payment-helper.ts

```diff
@@ -0,0 +1,28 @@
+import { SelectedFeeRecordsAvailablePaymentGroups } from '@ukef/dtfs2-common';
+
+export const getAvailablePaymentsHeading = (paymentGroups: SelectedFeeRecordsAvailablePaymentGroups) => {
+  if (!paymentGroups || paymentGroups.length === 0) {
+    throw new Error('No payment groups to select from.');
+  }
+
+  const numOfSinglePayments = paymentGroups.filter((group) => group.length === 1).length;
+  const numOfGroupPayments = paymentGroups.filter((group) => group.length > 1).length;
```

Comment:
I have a slightly pedantic preference to spell out the word number. I also wonder if these would be slightly clearer names:
`numberOfSinglePaymentGroups` and `numberOfMultiplePaymentGroups`
Also, the second one could just be the total number minus the first I think?